src/Microsoft.ML.Data/Scorers/PredictionTransformer.cs

```diff
@@ -20,39 +19,37 @@
 
 namespace Microsoft.ML.Runtime.Data
 {
-    public abstract class PredictionTransformerBase<TModel> : IPredictionTransformer<TModel>, ICanSaveModel
+
+    /// <summary>
+    /// Base class for transformers with no feature column, or more than one feature columns.
+    /// </summary>
+    /// <typeparam name="TModel"></typeparam>
+    public abstract class PredictionTransformerBase<TModel> : IPredictionTransformer<TModel>
```

Comment:
No documentation?

Reply:
it is there? Are you looking at iteration 10?

---
In reply to: [218905403](https://github.com/dotnet/machinelearning/pull/912#discussion_r218905403) [](ancestors = 218905403)